src/main/java/org/spongepowered/api/resource/ResourcePath.java

```diff
@@ -88,6 +88,9 @@ static ResourcePath parse(String path) throws IllegalArgumentException {
      */
     String getPath();
 
+    @Override
+    int hashCode();
```

Comment:
Why is this in the API?

Reply:
For arbitrary order?

Reply:
I put it there originally because I wanted it to be clear that this can be used in a Map. Now I'm just saying so in the docs.